content/pages/privacy-policy.md

```diff
@@ -0,0 +1,106 @@
+
+- [soundcloud.com](https://soundcloud.com)
+- [discogs.com](https://www.discogs.com)
+- [last.fm](https://www.last.fm)
+
+#### Broadcasting
+
+
+If you use the broadcasting feature of the Mixxx application, you need to enter login data of a third party Soundcast/Icecast server.
+Mixxx itself doesn't offer such a service.
+If used, the live audio stream as well as metadata of the played tracks is sent to these servers by the Mixxx application.
+
+
+#### Mixxx Crash Reporting
+
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports may be handled by the vendors of each platform (e.g., through the [Windows Error Reporting](https://learn.microsoft.com/windows/win32/wer/windows-error-reporting) feature).
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx core developers have access to these reports, which usually contain:
+
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+In addition Mixxx writes a `mixxx.log` file which contains entries that may expose personal data. In a support case we may ask to share this file in public. This is a manual process during which you may obfuscate sensitive date before sharing.  
+
+
+#### Mixxx Update Mechanism
+
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+In case updates are handled by the platform stores the user has downloaded Mixxx from,
+please refer to those platforms for information about the privacy policies.
+
+## 2. Main Mixxx Website
+
+The main website of Mixxx, which can be accessed via the `mixxx.org` domain, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment by Mixxx.
+
+
+## 3. Mixxx Community Websites
+
+The Mixxx community websites are related to development and community, and are the following:
+
+- GitHub (Code, Issue Tracking, Wiki)
+- Discourse Forum
+- Zulip Chat
+
+Each of those websites have different users systems. In order to participate users need to create a login, share some personal data and accept cookies. Refer the privacy statements of these sites for details:
+
+
+- GitHub Privacy Statement: [https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement](https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement)
+- Mixxx Discourse Forum Privacy Policy: [https://mixxx.discourse.group/privacy](https://mixxx.discourse.group/privacy)
+- Zulip Privacy Policy: [https://zulip.com/policies/privacy](https://zulip.com/policies/privacy)
+
+
+## Contact Us
+
+In case of privacy related question or requests, you can contact the `Mixxx Development Team` using the following e-mail address: ![Contact address for privacy issues]({static}/images/PrivacyAtMixxxDotOrg.png){align="top" height="25px" }

```

Comment:
Don't 'real' crawlers filter for ..at..dot.. strings?
I mean the image is probably some level of spam protection, but do we really need it? Other addresses are also written out and I didn't hear we have serious spam issue, do we @daschuer @ywwg ?
https://github.com/mixxxdj/website/blob/website/theme/templates/pages/contact.html#L12-L15
alt: encode with base64, or rot13 ;)

Reply:
Googles Spam Filter is pretty good, I don't think we need such countermeasures and can just write the address in plain text.